src/components/control/date.jsx

```diff
@@ -188,7 +188,7 @@ const Date = forwardRef(function Date(props, forwardedRef) {
         min={props.min}
         name={props.name}
         onChange={onInputChange}
-        onClick={onInputClick}
+        onMouseDown={onInputClick}
```

Comment:
I will to verify this. Mouse down could fire but the mouse up could occur on a different element (think click+hold). This could put the component in a weird state but it might not matter.